src/PixelArt/Singlarobin/VanillaJS/index.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Creates pixel art grid
+ * @param el DOM Element
+ * @param rows Number of rows
+ * @param rows Number of cols
+ */
+let selectedColor = '';
+let draggable = false;
+
+function PixelArt(el, rows, cols) {
+    // write logic to create pixel art grid.
+    const cellWidthString = getComputedStyle(
+        document.documentElement
+    ).getPropertyValue('--cell-width');
+
+    const cellWidthInt = parseInt(
+        cellWidthString.substring(1, cellWidthString.indexOf('p'))
+    );
+
+    const color = [];
+    for (let i = 0; i < rows; i++) {
+        color.push(`#${getColorValue(color)}`);
+    }
+    selectedColor = color[0];
+
+    const gridElement = document.querySelector(el);
+    gridElement.style.gridTemplateColumns = `repeat(${rows}, ${cellWidthString})`;
+    for (let i = 0; i <= rows; i++) {
+        for (let j = 0; j < cols; j++) {
+            const element = document.createElement('div');
+            element.id = `${i} ${j} `;
+            element.dataset.type = 'cell';
+            element.classList.add('cell');
+            gridElement.appendChild(element);
+            if (i === rows) {
+                element.classList.add('color');
+                element.style.background = color[j];
+            }
+        }
+    }
+
+    const width = rows * cellWidthInt;
+    const bottomBorder = document.createElement('div');
```

Comment:
why are we doing this? why can't we simply give border-bottom to grid using CSS?